### PR TITLE
fix: Operate ephemeral channel

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-3245](https://sendbird.atlassian.net/browse/UIKIT-3245)
[Ephemeral channel support - policy](https://sendbird.atlassian.net/wiki/spaces/UK/pages/2027325814/Ephemeral+channel+support)

## Description Of Changes

Ephemeral channel policy
* Group channel list
  * Remove the message receipt status (channel preview)
  * Remove the unread message count (channel preview)
* Group channel
  * Remove the message edit
  * Remove the message delete
  * Remove the message reactions
  * Remove the message receipt status (message)
  * Remove the message reply (quote_reply, thread)
* Group channel settings
  * Remove the search in channel
* Open channel
  * Remove the message edit
  * Remove the message delete

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [x] Improvement (refactor code)

[UIKIT-3245]: https://sendbird.atlassian.net/browse/UIKIT-3245?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/smart-components/ChannelList/components/ChannelPreview/index.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ interface ChannelPreviewInterface {
   channel: GroupChannel;
   isActive?: boolean;
   isTyping?: boolean;
+  isEphmeral?: boolean;
```
Comment (sravan-s) [inline src/smart-components/ChannelList/components/ChannelPreview/index.tsx:30]: dont need to drill down this prop, channel is already present as a prop, no?

Comment (HoonBaek) [reply src/smart-components/ChannelList/components/ChannelPreview/index.tsx:30]: Ah, it's a good catch, and why there's also a typo 😞🙀 my bad..

File: scripts/index_d_ts [hunk 2]
```diff
@@ -1060,9 +1060,9 @@ declare module "SendbirdUIKitGlobal" {
 
   export interface ThreadMessageInputProps {
     className?: string;
-    renderFileUploadIcon?: () =>  React.ReactElement;
-    renderVoiceMessageIcon?: () =>  React.ReactElement;
-    renderSendMessageIcon?: () =>  React.ReactElement;
+    renderFileUploadIcon?: () => React.ReactElement;
```
Comment (sravan-s) [inline scripts/index_d_ts:1063]: 👍

File: scripts/index_d_ts [hunk 3]
```diff
@@ -2248,8 +2250,10 @@ declare module '@sendbird/uikit-react/ui/OpenChannelFileMessage' {
 
 declare module '@sendbird/uikit-react/ui/OpenChannelOGMessage' {
   import type SenbirdUIKitGlobal from 'SendbirdUIKitGlobal';
-  interface OOpenChannelOGMessageProps {
+  interface OpenChannelOGMessageProps {
```
Comment (sravan-s) [inline scripts/index_d_ts:2253]: 👍 thanks

**Review by sravan-s — APPROVED** [review]

**Pull request merged**